task_DEV-11/FileReader.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace task_DEV_11
+{
+  public class FileReader
+  {
+    public Dictionary<string, string> GetAlphabetFromFile(string filePath)
+    {
+      // Get alphabet from file.
+      string[] alphabetFromFileFormantPairs = null;
+      using (TextReader textReader = new StreamReader(filePath))
+      {
+        char[] delimiters = { '[', ']', '\r', '\n', '\t' };
+        alphabetFromFileFormantPairs = textReader.ReadToEnd()
+          .Split(delimiters, StringSplitOptions.RemoveEmptyEntries);
+      }
+
+      // Add it to the dictionary.
+      var alphabet = new Dictionary<string, string>();
+      foreach (var formantsMapping in alphabetFromFileFormantPairs)
+      {
+        char[] delimiters = { '-' };

```

Comment:
You have already delimiters. use the same name to overload the previous one is a very bad idea